crates/stages/src/stages/index_account_history.rs

```diff
@@ -217,7 +241,7 @@ mod tests {
 
         // verify
         let table = cast(db.table::<tables::AccountsHistory>().unwrap());
-        assert_eq!(table, BTreeMap::from([(shard(u64::MAX), vec![1, 2, 3])]));
+        assert_eq!(table, BTreeMap::from([(shard(u64::MAX), vec![0, 1, 2, 3])]));
```

Comment:
unless i'm misunderstanding, we do insert a changeset at block 0, so the history should also have that.